.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,24 @@
+# This is a workflow to run the deploy snapshot pipeline in Azure DevOps
+name: pr_validation
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the dev branch
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+    branches: [ dev ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # Make sure we have a changelog entry
+  changelog:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dangoslen/changelog-enforcer@v1.4.0
```

Comment:
Have we got (do we need) an approval for this?

Reply:
The [OSS guide](https://docs.opensource.microsoft.com/content/using/index.html) seems to be focusing on open source components that are part of the product only. I'd suggest reaching out to them to see if there's anything we have to do in order to use it as part of our build pipeline.

Reply:
There doesn't seem to be any guidance right now.  Eventually they'll end up needing to have a path for these, but right now there's nothing.  I'm thinking we use it now, and clone it if we need to.